build.gradle

```
@@ -13,7 +13,7 @@ targetCompatibility = 1.8
 
 group = 'net.fabricmc'
 archivesBaseName = project.name
-version = '0.2.6-SNAPSHOT'
+version = '0.2.7.014'
```

Lets get this into 0.2.6 as its still the "experimental" branch

```suggestion
version = '0.2.6-SNAPSHOT'
```